### DISCUSSION TITLE
Bind buttons to a certain command/action

### DISCUSSION THREAD

**Discussion by TRaSH-** [pannal/plex-for-kodi#138] [Ideas]
Would be nice if we would have the option to bind more buttons to a certain command. 

In my case

- Stream Info
- subtitledelay

But I think some people would also like the option for:

- audio language
- subtitle change

**Comment by pannal** [comment]
What would the last three bind to?

* subtitledelay: open the delay adjustment immediately
* audio language: open the dialog, or instantly swap to the next audio stream?
* subtitle change: same question

**Comment by TRaSH-** [comment]
subtitledelay: open the delay adjustment immediately
audio language: Because some media files can have a lot of audio langues I think  => open the dialog, so you can choose
subtitle change: same as with audio

**Reply by pannal** [reply]
Sounds good

**Comment by pannal** [comment]
This feature is on my "short" list.
